src/Authentication/App/Controllers/LoginController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Authentication\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Authentication\App\Requests\LoginRequest;
+use Lightit\Authentication\App\Rescources\LoginResource;
+use Lightit\Authentication\Domain\Actions\LoginAction;
+use Lightit\Authentication\Domain\DataTransferObjects\LoginInputDto;
+
+class LoginController
+{
+    public function __invoke(LoginRequest $request, LoginAction $loginAction): JsonResponse
+    {
+        $dto = new LoginInputDto(
+            email: LoginRequest::EMAIL,
+            password: LoginRequest::PASSWORD
+        );
```

Comment:
This code is equal to:
```php
$dto = new LoginInputDto(
    email: 'email',
    password: 'password'
);
```
You're accessing the class' constants, not the request's payload values. This should be replaced to:
```suggestion
        $dto = new LoginInputDto(
            email: $request->string(LoginRequest::EMAIL)->toString(),
            password: $request->string(LoginRequest::PASSWORD)->toString()
        );
```
Or better yet:
```php
// In LoginRequest.php...
public function toDto(): LoginInputDto
{
    return new LoginInputDto(
        email: $this->string(LoginRequest::EMAIL)->toString(),
        password: $this->string(LoginRequest::PASSWORD)->toString()
    );
}

// In LoginController.php...
$dto = $request->toDto();
```